low-level/sensors/impl/platforms/realstor/realstor_logical_volume_sensor.py

```diff
@@ -48,7 +48,12 @@ class RealStorLogicalVolumeSensor(SensorThread, InternalMsgQ):
     SENSOR_NAME = "RealStorLogicalVolumeSensor"
     SENSOR_RESP_TYPE = "enclosure_logical_volume_alert"
     RESOURCE_CATEGORY = "cortx"
-    RESOURCE_TYPE = "enclosure:cortx:logical_volume"
+    RESOURCE_TYPE_LVOL = "enclosure:cortx:logical_volume"
+    RESOURCE_TYPE_DG = "enclosure:cortx:disk_group"
```

Comment:
rename file to reflect both resources being monitored - diskgroup & volume. Grouping them together in one sensor is good for code re-usability, unlike other realstor sensors.
realstor_dg_volume_sensor.py